airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -942,6 +968,15 @@ def __init__(self,
         self.schema_object = schema_object
 
         # BQ config
+        if not table_resource:
+            warnings.warn(
+                "Passing table parameters via key words arguments will be deprecated. "
```

Comment:
By when will it be deprecated ?

Reply:
No idea, we've never discussed when we will deprecate such things